packages/mdc-slider/mdc-slider.scss

```diff
@@ -27,22 +26,13 @@
   @include mdc-slider-color-accessible(secondary);
 
   &--disabled {
-    $light-color: #9a9a9a;
-    $dark-color: #787878;
+    $disabled-color: #9a9a9a;
 
-    @include mdc-slider-highlight-color_($light-color);
-    @include mdc-slider-rail-color_($light-color);
-    @include mdc-slider-rail-tick-mark-color_($light-color);
-    @include mdc-slider-thumb-color_($light-color);
+    @include mdc-slider-highlight-color_($disabled-color);
+    @include mdc-slider-rail-color_($disabled-color);
+    @include mdc-slider-rail-tick-mark-color_($disabled-color);
+    @include mdc-slider-thumb-color_($disabled-color);
     @include mdc-slider-thumb-stroke-cutout_($mdc-slider-default-assumed-bg-color);
```

Comment:
We can remove the `$mdc-slider-dark-theme-assumed-bg-color` variable from `_variables.scss`